CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.29.4] - 2022-03-09
+
+### Fixed
+
+- When executing a task on an alternate Conda environment, Covalent does not have to be installed on that environment.
```

Comment:
Can you also add a brief explanation here that how exactly that is done. Like now we're deserializing the `TransportableObject` before passing it to the conda env.

Reply:
Done